code/modules/projectiles/magazines/specialist.dm

```diff
@@ -88,7 +88,7 @@
 	name = "\improper generic high-explosive rocket"
 	desc = "A precursor to all kinds of rocket ammo unfit for normal use. How did you get this anyway?"
 	caliber = CALIBER_84MM
-	icon_state = "rocket"
```

Comment:
wait are players getting this? they shouldnt be

Reply:
In default gameplay, they don't get it at all, however it's avviable in the EORD areas and *rarely* normally, funnly enough i tested it and it works just fine.

Reply:
Yea this sounds like it shouldnt be happening honestly, I consider that a "dont spawn the thing you're not supposed to spawn" issue rather than a sprite one

Reply:
Then there's an inconsistency regarding icons, since some ammo types have icons, but other generic ones don't.